### PR TITLE
Beta release!

### PR DESCRIPTION
**Pull request by JonEsparaz**
First working version! I expect there will be a small number of bugs, but it should be usable for a "regular" competition.

I've tested this out with groups I generated for [Appleby Summer A 2023](https://www.worldcubeassociation.org/competitions/ApplebySummerA2023) using Delegate Dashboard.

Problems I would like to solve:
- The PDF rendering is _really_ slow. I suspect this has to do with the scorecards being (poorly) defined in terms of fixed pixels, so the PDF generator needs to scale the HTML. I might need to optimize the images too.
- UTF-8 encoded characters aren't supported by the PDF rendered (we are currently stripping those characters from names). Need to figure out the font situation.
- The application's design is kinda whack (almost all of the React states live in `App.tsx`, which creates an annoying amount of prop drilling and unnecessary re-rendering of the application). Additionally, I'm almost certain that my algorithms and data structures can be improved.

Features I would like to add:
- Currently, you can't configure "groups" for a round that doesn't have them. For example, if we wanted to pre-print scorecards for a top-75% 3x3x3 Round 2, we wouldn't be able to chunk them in "groups".

### CODE REVIEW

File: src/scorecards/AverageOfFive.tsx
Comment (kr-matthews) [inline src/scorecards/AverageOfFive.tsx:1]: A lot of the table code is repeated a lot in various files.

File: src/scorecards/helper.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  }
+
+  return null;
+};
+
+export const getTimeLimitText = (
+  timeLimit: Round["timeLimit"] | null | undefined,
+) => {
+  if (timeLimit && Array.isArray(timeLimit.cumulativeRoundIds)) {
+    if (timeLimit.cumulativeRoundIds.length === 1) {
```
Comment (kr-matthews) [inline src/scorecards/helper.ts:49]: I think we need to check that the length is either 0 or 1.
Edit: you fixed this already I guess.

**Review by kr-matthews — COMMENTED** [review]
Nice.

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { FooterHelp, Link } from "@shopify/polaris";
+
+export const Footer = () => (
+  <FooterHelp>
+    Learn more about this project on{" "}
+    <Link
+      url="https://github.com/Speedcubing-Canada/strong-scorecards"
```
[the remark this answers is not in the document]
Comment (JonEsparaz) [reply src/components/Footer.tsx:7]: Good catch

File: src/reportWebVitals.ts
[the remark this answers is not in the document]
Comment (JonEsparaz) [reply src/reportWebVitals.ts:1]: Probably not. It just comes with CRA. Will remove 👍🏼

File: src/scorecards/AverageOfFive.tsx
Comment (JonEsparaz) [reply src/scorecards/AverageOfFive.tsx:1]: Leaving this for later: https://github.com/Speedcubing-Canada/sarahs-scorecards/issues/3

**Comment by JonEsparaz** [comment]
Merging in as this isn't meant to be perfect. 

Smaller PRs with improvements/fixes from me (or hopefully other contributors!) to follow.

**Pull request merged**